app/views/layouts/_navigation.html.erb

```diff
@@ -1,14 +1,23 @@
+<!-- NAVIGATION -->
+<div id="menu">
+	<nav class="navbar-wrapper navbar-default">
+		<div class="container">
+			  <div class="navbar-header">
```

Comment:
Classes should not include (.)Dot.
Change `.navbar-backyard` to `navbar-backyard`